lib/tag/vdom.js

```diff
@@ -38,10 +38,11 @@ function mountTo(root, tagName, opts) {
 
 }
 
-riot.tag = function(name, html, css, fn) {
+riot.tag = function(name, html, css, attrs, fn) {
+  if (typeof attrs == 'function') {fn = attrs; attrs=css}
```

Comment:
The compiled tag before v2.0.15 has just 4 args on `riot.tag()`, so `attrs` will contain the content of `css` with your original code. It makes error when the `attrs` is analyzed. Thanks.

```javascript
riot.tag('my-tag', '<my-tag >HELLO!</my-tag>', 'my-tag, [riot-tag="my-tag"]  { color: red }', function(opts) {
    // some code
});
```